packages/graphql/src/graphql/objects/DeleteInfo.ts

```diff
@@ -21,6 +21,7 @@ import { GraphQLInt, GraphQLNonNull, GraphQLObjectType, GraphQLString } from "gr
 
 export const DeleteInfo = new GraphQLObjectType({
     name: "DeleteInfo",
+    description: "Information about the deletion of a node or relationship.",
```

Comment:
I think a more accurate description would be "Information about the number of nodes and relationships deleted during a delete mutation".